src/foam/u2/TableView.js

```diff
@@ -46,6 +49,14 @@ foam.CLASS({
   methods: [
     function addObj(obj) {
       var e = this.start('tr')
+          .on('click', function() {
+            this.tableView.selection = obj;
```

Comment:
I wouldn't hold up the CL for this, but add a TODO to try using a single listener for performance/memory.